testdata/baselines/reference/fourslash/findAllReferences/jsxFindAllReferencesOnRuntimeImportWithPaths1.baseline.jsonc

```diff
@@ -0,0 +1,3 @@
+// === findAllReferences ===
```

Comment:
Not sure if this is relevant for this PR, but this baseline (and the `tslibFindAllReferences...` below) still looks like it's missing other references.

Reply:
I have more code I could push which scaffolds more of this out; let me see if that (questionable) code works for this